docs/detections/osquery-response-action.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[osquery-response-action]]
+=== Add Osquery Response Actions
+Osquery Response Actions allow you to add live queries to custom query rules so you can automatically collect data on systems the rule is monitoring. Use this data to support your alert triage and investigation efforts.
+
+NOTE: Response Actions are in technical preview.
+
+[role="screenshot"]
+image::images/available-response-actions.png[Shows available response actions]
+
+.Requirements
+[sidebar]
+--
+* Osquery Response Actions are available with a https://www.elastic.co/pricing[Platinum or Enterprise subscription].
+* The {kibana-ref}/manage-osquery-integration.html[Osquery manager integration] must be installed.
+* {agent}'s {fleet-guide}/view-elastic-agent-status.html[status] must be `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* Your role must have {kibana-ref}/osquery.html[Osquery feature privileges].
+--
+
+[float]
+[[add-osquery-response-action]]
+=== Add Osquery Response Actions to rules
+
+You can add Osquery Response Actions to new or existing custom query rules. Queries are ran every time the rule executes.
```

Comment:
Or just "Queries run every time" to avoid passive voice
```suggestion
You can add Osquery Response Actions to new or existing custom query rules. Queries are run every time the rule executes.
```